apps/src/music/player/sound.js

```diff
@@ -55,17 +55,25 @@ export function PlaySound(
   groupTag,
   when = 0,
   onStop = () => {},
-  loop = false
+  loop = false,
+  effects = false
 ) {
   for (var i = 0; i < soundList.length; i++) {
     if (soundList[i] === name) {
       // Always provide a groupTag.  If one wasn't provided, just use the sound name as the group name.
-      return PlaySoundByIndex(i, groupTag || name, when, loop, onStop);
+      return PlaySoundByIndex(i, groupTag || name, when, loop, effects, onStop);
     }
   }
 }
 
-function PlaySoundByIndex(audioBufferIndex, groupTag, when, loop, onStop) {
+function PlaySoundByIndex(
+  audioBufferIndex,
+  groupTag,
+  when,
+  loop,
```

Comment:
Do we still need the `loop` param? Are we using it anywhere? Also definitely agree that we should start using objects to pass params here as the list grows larger 🙂 (probably as a follow-up)

Reply:
We've never used it, but it came for free with the original audio code I donated. :). I might leave it in for a bit longer just in case a need arises soon, but it could probably go away.